doc/JSON_FLAGS.md

```diff
@@ -703,6 +707,7 @@ List of known flags, used in both `terrain.json` and `furniture.json`.
 - ```mx_portal_in``` ... Another portal to neither space.
 - ```mx_portal``` ... Portal to neither space.
 - ```mx_roadblock``` ... Roadblock furniture with turrets and some cars.
+- ```mx_roadworks``` ... Partialy closed damaged road with chance of work equipment and utility vehicles.
```

Comment:
While you're here, could you please also add docs for bandits roadblock that I've forgot to add?